src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -30,7 +30,7 @@ const GordonSchedulePanel = ({ profile, myProf }: Props) => {
   const [selectedSchedule, setSelectedSchedule] = useState<Schedule | null>(null);
   const [selectedCourse, setSelectedCourse] = useState<CourseEvent | null>(null);
   const [isScheduleOpen, setIsScheduleOpen] = useState<boolean>(
-    localStorage.getItem(scheduleOpenKey) === 'true' ?? true,
+    localStorage.getItem(scheduleOpenKey) !== 'true',
```

Comment:
I believe this is backwards. Using `!== 'true'` will cause `isScheduleOpen` to default to `false` when the localStorage item `profile.schedule.isOpen` equals `'true'` (meaning the schedule will be closed), and `isScheduleOpen` will default to `false` when `profile.schedule.isOpen` equals `'false'`. Here are screenshots showing this behavior:
The schedule defaulted open when the local storage value is 'true'
![image](https://github.com/gordon-cs/gordon-360-ui/assets/35319956/56565d78-7548-48f1-81a0-bf1fa45655c2)
The schedule defaulted closed when the local storage value is 'false'
![image](https://github.com/gordon-cs/gordon-360-ui/assets/35319956/28c38da2-214f-4761-8815-71bf55253f9e)

We want instead to check whether the local storage value is equal to the string `'false'`:
```suggestion
    localStorage.getItem(scheduleOpenKey) !== 'false',
```
That will evaluate to the boolean value `false` only when the local storage value has been set to the string `'false'`. If the local storage value is set to the string `'true'`, or if there is no value in local storage, this will evaluate to the boolean value `true`, meaning the schedule will be open.

Reply:
Yes, this seems to work, thanks again.